src/views/projects/Cob/CobHeader.svelte

```diff
@@ -67,7 +68,11 @@
         <TextInput transparent variant="form" bind:value={title} />
       {:else}
         {#if title}
-          <span class="txt-medium title">{title}</span>
+          <div class="title">
+            <InlineMarkdown
+              --font-size="var(--font-size-medium)"
```

Comment:
Isn't there any way to type this prop?
I tried some things but wasn't successful..

Reply:
Not really. We could encapsulate this in a regular prop, like we do elsewhere.

E.g.

`export let size: "small" | "regular" = "regular"`

And then just add the usual CSS in the component.

Reply:
Will do that, so I don't introduce another way of doing things that's not aligned with the rest.

Reply:
cc2aab77926cb423f3b70df39cda9a3e90a44196

Reply:
Nice, I like the idea of passing the variables directly, but without type safety I feel like we're introducing new potential regressions.

Reply:
Hmmmmmm, just realised that it's not the same thing actually.
When we pass down the CSS variable, we use it in the `<script>` section with the `:global(code)` modifier. So basically it's not possible to achieve the same effect. Because applying it to the whole element changes the size of all the text.

Reply:
<img width="602" alt="Screenshot 2023-04-27 at 16 55 21" src="https://user-images.githubusercontent.com/158411/234902662-2f328f08-2e84-4e2a-a5a0-6a6b491572f4.png">
<img width="527" alt="Screenshot 2023-04-27 at 16 56 10" src="https://user-images.githubusercontent.com/158411/234902670-4c63807d-bfad-4178-ba84-6b190d916e6c.png">

Reply:
Ok gotcha, I'm okay with 175b513 will be on the lookout if something changes with typed props to support CSS variables maybe we can circle back here